--- a/CMakeLists.txt	2026-05-09 13:29:42.093013443 +0200
+++ b/CMakeLists.txt	2026-05-09 13:31:59.701652866 +0200
@@ -2,13 +2,6 @@
 # Top-level CMakeLists.txt file for Crazy Eddie's GUI System
 ################################################################################
 cmake_minimum_required(VERSION 2.8.11)
-if (POLICY CMP0017)
-    cmake_policy(SET CMP0017 OLD)
-endif()
-
-if (POLICY CMP0045)
-    cmake_policy(SET CMP0045 OLD)
-endif()
 
 if (APPLE AND NOT CMAKE_OSX_ARCHITECTURES)
     set( CMAKE_OSX_ARCHITECTURES x86_64 CACHE STRING "Build architectures for Mac OS X")
